packages/angular/package.json

```diff
@@ -13,15 +13,12 @@
     "serve:dark": "ng serve dev -c dark",
     "storybook": "start-storybook -s .storybook/public -p 6006",
     "storybook:build": "build-storybook -s .storybook/public -c .storybook -o ../../dist/website/storybook/angular --quiet",
-    "postinstall": "ngcc && node ../../scripts/ngcc-clean",
-    "build": "npm-run-all build:lib:cds build:lib:clr build:package:cds build:package:clr build:update:cds build:update:clr",
+    "build": "npm-run-all build:lib:cds build:lib:clr build:update:cds build:update:clr",
     "build:preinstall": "yarn run build:preinstall:cds && yarn run build:preinstall:clr",
-    "build:preinstall:cds": "mkdir -p ../../dist/angular/",
-    "build:preinstall:clr": "mkdir -p ../../dist/clr-angular/",
-    "build:package:clr": "replace '@VERSION' $npm_package_version ./dist/clr-angular/package.json",
-    "build:package:cds": "replace '@VERSION' $npm_package_version ./dist/cds-angular/package.json",
+    "build:preinstall:cds": "mkdir -p ../../dist/angular/ && cp ./projects/cds-angular/package.json ../../dist/angular/",
+    "build:preinstall:clr": "mkdir -p ../../dist/clr-angular/ && cp ./projects/clr-angular/package.json ../../dist/clr-angular/",
     "build:update:clr": "node ../../scripts/update-local-packages.js /packages/angular/dist/clr-angular/package.json",
-    "build:update:cds": "node ../../scripts/update-local-packages.js /packages/angular/dist/cds-angular/package.json",
+    "build:update:cds": "node ../../scripts/update-local-packages.js /packages/angular/dist/angular/package.json",
```

Comment:
Updated packages to copy the package json so yarn would properly resolve all dependencies at top level. It was missing packages during install since the package.json was missing.